.github/workflows/lint.yml

```diff
@@ -105,6 +105,9 @@ jobs:
         with:
           toolchain: ${{ vars.RUST_VERSION_API_DIFF }}
 
+      - name: Add nightly toolchain for cargo-public-api
+        run: rustup install nightly --profile minimal
+
```

Comment:
installing rust is done above

Reply:
It seems that cargo public api requires `nightly-2024-10-18`

Reply:
Hum, it's strange, it seems to install stable version on every PR since https://github.com/actix/actix-web/pull/3501 var seems to be empty, maybe as user we don't have access to this variable so it's empty by default for us (but not for you ?).